src/services/school/model.js

```diff
@@ -63,7 +63,7 @@ const schoolSchema = new Schema(
 			default: '',
 			enum: ['', 'school', 'schoolGroup'],
 		},
-		officialSchoolNumber: { type: String },
+		officialSchoolNumber: { type: String }, // TODO unique per federal state validation on save
```

Comment:
add unique index for {federalstate:1, officialSchoolNumber:1} after https://github.com/hpi-schul-cloud/schulcloud-server/pull/3224/files#diff-91fedf68c25ac4a358983fd4ee659f3a6a4d642725b3f864f06a3bc285c168e1R110